Comparator.dcm

```diff
@@ -90,10 +90,16 @@ K dual comparator
 F http://www.ti.com/lit/ds/symlink/lmv331.pdf
 $ENDCMP
 #
-$CMP LMV7219
-D Single Low-Power High-Speed Push-Pull Output Comparator, SOT-23-5/SC-70
+$CMP LMV7219M5
+D Single Low-Power High-Speed Push-Pull Output Comparator, SOT-23-5
 K single comparator
-F http://www.ti.com/lit/ds/symlink/lmv7219.pdf
+F https://www.ti.com/lit/ds/symlink/lmv7219.pdf
+$ENDCMP
+#
+$CMP LMV7219M7
+D Low-Power High-Speed Push-Pull Output Comparator, SC-70-5
```

Comment:
Prepend `Single` for consistency

Reply:
Huh, the aliased component (MCP6561) was also missing this. I added it to that one too.

Btw, do you want me to merge, rebase or even squash my changes?

Reply:
No! Please don't do anything altering git history during a review.